### PR TITLE
Update raspibolt_40_lnd.md (many updates of the lnd/litd guide)

### LINKED ISSUE #748: LND configuration chapter small issue
A small issue while trying to set up lnd.conf
On line 10 of the lnd.conf I got any error:
/home/bitcoin/.lnd/lnd.conf:10: unknown option: protocol.anchors
When checking sample lnd conf files it seems that you need to use protocol.**no-anchors**=true
Then lnd can configure correctly

### PR DESCRIPTION
**Pull request by VajraOfIndra**
Summary: the updates aim at:
1. using the latest lnd and litd releases,
2. correcting typos and broken links,
3. adding useful options to the lnd/litd configuration files,
4. replacing outdated commands and examples and warning for outdated videos, and;
5. adding a few safety information for some commands.

The full list of changes and rationales for them:
1. Added warning above most videos: (Warning: the video below is outdated and does not correspond exactly to the commands above)
2. Updated the 'download and install' section with latest release (v0.13.1). Removed bitconner-related lines as not used for the verification process.
3. Updated lnd.conf content 
4. Removed protocol.anchor=true (now by default) (Fixes #748 )
5. Added AMP payments options (did not add keysend as AMP renders keysend obsolete with time)
6. Made minchansize option visible and changed it from default (20,000) to 100,000 sats
7. Made base fee and fee rate settings visible (kept default value)
8. Modified the default value of coop-close-target-confs to save on closing fees
9. Added the startup speed section from https://www.lightningnode.info/advanced-tools/lnd.conf (by Openoms)
10. Removed the 'sudo su - bitcoin' user login at the start of section "Run LND": not needed as we already are with user bitcoin after the lnd.conf section.
11. Removed "and admin.macaroon" when veryfing the TLS certs in folder /mnt/ext/lnd/
12. Added commands to allow safe reboots of the node (line 373)
13. Replaced the np2wkh address creation for funding by a bech32 p2wkh address
14. Replaced the nested segwit full address by the string bc1... (to avid risk of funds being sent to example address)
15. Added 'optional' to the autopilot mention as it is not by default
16. Added amboss.space together with 1ML as amboss despite being newer seems to be more actively developped than 1ml.
17. Default node to demonstrate lncli connect and lncli openchannel is down, replaced by ACINQ which should be a stable long-lasting node and useful for routing.
18. Removed the lncli openchannel command that does not use the optional fee selection --sat_per_byte (leads to needlessly overpaying fees)
19. Replaced deprecated sat_per_byte with preferred sat_per_vbyte in the openchannel command
20. Broken link converter satoshi to USD, fixed
21. Add comment and link to check mempool.space to select appropriate fee
22. In "More commands", added an example on how to send an invoice-less payment to a node using AMP
23. Added sat_per_vbyte option to cloeschannel command
24. Ask for selection of a new passwod [E] for the Lightning Terminal UI password
25. Change LiT password to a new password [E] rather than reusing RPC password [B]
26. Updated lit.conf to reflect changes above and added options for Pool and Faraday

### CODE REVIEW

File: raspibolt_40_lnd.md [hunk 1]
```diff
@@ -29,38 +29,34 @@ Download and install LND
...
-> Primary key fingerprint: 9C8D 6186 8A7C 4920 03B2  744E E7D7 37B6 7FA5 92C7
-$ gpg --verify roasbeef-manifest-v0.11.1-beta.txt.sig manifest-v0.11.1-beta.txt
-> gpg: Good signature from "Olaoluwa Osuntokun <laolu32@gmail.com>" [expired]
-> gpg: Note: This key has expired!
-> Primary key fingerprint: 9769 140D 255C 759B 1EB7  7B46 A963 87A5 7CAA E94D
->      Subkey fingerprint: 4AB7 F8DA 6FAE BB3B 70B1  F903 BC13 F65E 2DC8 4465
-
-$ tar -xzf lnd-linux-armv7-v0.11.1-beta.tar.gz
-$ sudo install -m 0755 -o root -g root -t /usr/local/bin lnd-linux-armv7-v0.11.1-beta/*
+>gpg: Signature made Mon 19 Jul 2021 22:41:37 BST
```
Comment (vicariousdrama) [inline raspibolt_40_lnd.md:45]: Should this have the following
`gpg --verify manifest-roasbeef-v0.13.1-beta.sig manifest-v0.13.1-beta.txt`

File: raspibolt_40_lnd.md [hunk 2]
```diff
@@ -97,8 +93,57 @@ Now that LND is installed, we need to configure it to work with Bitcoin Core and
...
+  # Whether the databases used within lnd should automatically be compacted on
+  # every startup (and if the database has the configured minimum age). This is
+  # disabled by default because it requires additional disk space to be available
+  # during the compaction that is freed afterwards. In general compaction leads to
+  # smaller database files.
+  db.bolt.auto-compact=true
+  # How long ago the last compaction of a database file must be for it to be
+  # considered for auto compaction again. Can be set to 0 to compact on every
+  # startup. (default: 168h)
+  db.bolt.auto-compact-min-age=168
```
Comment (vicariousdrama) [inline raspibolt_40_lnd.md:146]: The value for this property should express the time unit. e.g. `168h`

File: raspibolt_40_lnd.md (hunk 1 above)
Comment (VajraOfIndra) [reply raspibolt_40_lnd.md:46]: Thanks! I added the missing the command.

File: raspibolt_40_lnd.md (hunk 2 above)
Comment (VajraOfIndra) [reply raspibolt_40_lnd.md]: Thanks. I couldn't find information about time units for this property, do you know where it is described?
I fixed the value by adding the time unit and also added a bit of explanation about the time format. For both the lnd.conf and litd.conf.

**Comment by Stadicus** [comment]
Great update, thanks a lot!

**Comment by VajraOfIndra** [comment]
Great to see it merged, thanks! :)

**Pull request merged**